EnglishFilter/sections/css_extended.txt

```diff
@@ -1907,6 +1907,7 @@ indiastudychannel.com#?##content_rightbar > div.roundedBlock:has(> div[id^="div-
 inenglishwithlove.com#?#.sqs-block:has(> div.sqs-block-content div.image-button-inner > a[href^="https://elsaspeak.com/"])
 couponxoo.com#?#body > div.container:has(> p.text-center:contains(Advertisement))
 couponxoo.com#?#.row > div:has(> ins.adsbygoogle)
+presearch.org#?#div[x-data="searchResults()"] div[x-data]:has(> div > div > div[class] > span:contains(Ad))
```

Comment:
It is possible to hide without extended CSS(also context ad may be unblocked in useful ads filter)
<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/8361299/165907465-c42fbd03-1fe9-4997-98f7-435778c5b0c3.png)

</details>